spec/requests/acuant_sdk_spec.rb

```diff
@@ -32,29 +37,28 @@
         end
       end
 
-      wasm_js = "#{base_url}/AcuantImageProcessingWorker.wasm"
+      wasm_js = base_url.start_with?('/acuant/') ?
+        "#{base_url}/AcuantImageService.wasm" :
+        "#{base_url}/AcuantImageProcessingWorker.wasm"
       context wasm_js do
         before { get wasm_js }
 
         it 'renders a WASM asset' do
           expect(response.status).to eq(200)
           expect(response.headers['Content-Type']).to eq('application/wasm')
           expect(response.body.length).to eq(
-            File.size('public/acuant/11.4.3/AcuantImageProcessingWorker.wasm'),
+            base_url.start_with?('/acuant/') ?
+              File.size('public/acuant/11.5.0/AcuantImageService.wasm') :
+              File.size('public/acuant/11.4.3/AcuantImageProcessingWorker.wasm'),
           )
         end
 
-        it 'includes a CSP header with unsafe-eval' do
-          expect(response.headers['Content-Security-Policy']).
-            to match(/script-src [^;]*'unsafe-eval'/)
-        end
-
```

Comment:
This is another case of LG-3785 biting us: The `unsafe-eval` doesn't actually exist anywhere other than in development and test environments:

https://github.com/18F/identity-idp/blob/b08d237ef44846efbba33f6a3de1d56ab117b645/config/initializers/secure_headers.rb#L47-L51

It may have been aimed at this directive, though the test wasn't actually exercising it, and it's no longer necessary from my initial testing:

https://github.com/18F/identity-idp/blob/b08d237ef44846efbba33f6a3de1d56ab117b645/app/controllers/concerns/idv/document_capture_concern.rb#L8-L9

Reply:
> it's no longer necessary from my initial testing:

Evidently, it is still needed, at least for Android Chrome, which was failing to load in testing without the `unsafe-eval`. Restored in 958181f.